### ISSUE TITLE
search field and group by type

### ISSUE THREAD

**Issue by Danieleeee** [patrys/vscode-code-outline#3]
It would be useful to have a field for filtering Function, Metod,...

It would be nice to group items by type, like **_@:_** in symbol search.

Thaks.

**Comment by patrys** [comment]
Items are currently grouped by type and sorted alphabetically within the same type.

Unfortunately there's currently no API in vscode to implement a search field but I think `@` in symbol search fulfills that purpose nicely.

**Comment by Danieleeee** [comment]
> Items are currently grouped by type and sorted alphabetically within the same type.

Is possible to allow the user to sort alphabetically or not?

**Comment by tlawrenz** [comment]
Is there still no API for a search field? 
(As now the outline view has moved to a new sidebar (#70))

**Comment by patrys** [comment]
No, unfortunately not. However it's possible that VS Code will gain its own symbol tree in the upcoming weeks.
